backend/dataall/core/stacks/db/enums.py

```diff
@@ -6,7 +6,7 @@ class StackStatus(Enum):
     CREATE_COMPLETE = 'CREATE_COMPLETE'
     CREATE_FAILED = 'CREATE_FAILED'
     DELETE_IN_PROGRESS = 'DELETE_IN_PROGRESS'
-    DELETE_COMPLETE = 'DELETE_FAILED'
```

Comment:
I spotted this bug while debugging something else. Hence fixed it here. Please let me know if it looks good, otherwise I can remove this change!

Reply:
It is very minor, we can include it here